ens/main.py

```diff
@@ -29,7 +30,12 @@
     raw_name_to_hash,
 )
 
-ENS_MAINNET_ADDR = '0x314159265dD8dbb310642f98f50C066173C1259b'
+NET_VERSION_TO_ENS_ADDR = {
+    1: '0x314159265dD8dbb310642f98f50C066173C1259b',
```

Comment:
Actually, sorry to keep nitpicking here but it would be nice if there was something that made it clear what chain id belonged to which network.  Here are some options:
```python
NET_VERSION_TO_ENS_ADDR = {
    1: ..., # Mainnet
    ...
}
```
-or-
```python
MAINNET_NET_VERSION = 1
...
NET_VERSION_TO_ENS_ADDR = {
    MAINNET_NET_VERSION: ...,
}
```
The second option seems a bit nicer since it gives us other constants that could be useful.  It's just as verbose as the code before my original suggestion, but it still has the advantage of using `dict` methods instead of re-implementing `dict`-like functionality in a custom function.

Incidentally, it looks like there's already an `ens.constants` module that might be a better place to stick these values.  Then you'd just import them in this module.

I want to CC @pipermerriam and @carver here and ask what's the preferred terminology we should use here: "net version" or "chain id"?  Would "chain id" actually be a better term to represent in the names of these constants?

Reply:
I also updated this.

Regarding "net version" or "chain id" I don't know what to use. Some documentation says net version and some chain id. If needed, I can also rename this.

Reply:
> Actually, sorry to keep nitpicking here

Yeah, I would say let's get this PR into a basic working state, make sure the high-level arch is okay, and then get into the nits.

> I want to CC @pipermerriam and @carver here and ask what's the preferred terminology we should be using: "net version" or "chain id"? Would "chain id" actually be a better term to represent in the names of these constants?

Presumably, it could even be the combination that's important: a pair of network and chain ID might actually be the right index.